tests/selenium/sequential-paypal-test.js

```diff
@@ -1,26 +1,24 @@
-module.exports = function(driver, By) {
-  return function(done) {
-    driver.get('http://localhost:3000/en-US/');
-    driver.findElement(By.id('amount-other-input')).sendKeys('10');
-    driver.findElement(By.css('.page-active .next-button')).click();
-    driver.findElement(By.css('.paypal-button .payment-submit-button')).click();
-    driver.wait(function() {
-      return driver.getCurrentUrl().then(function(url) {
-        return url.indexOf('https://www.sandbox.paypal.com/cgi-bin/webscr') === 0;
-      });
+module.exports = function(driver, By, done) {
```

Comment:
This is because we no longer store the context for driver and by in a separate scope from where done comes from.

Because they all come from the same scope, I can pass them all in one shot, thus only need one function.